### DISCUSSION TITLE
lotus depolama madencisi hakkında

### DISCUSSION THREAD

**Discussion by BurakErcan19** [filecoin-project/lotus#6801] [General Q&A]
bu işi türkiyede yapan yok ve ingilizce dilimiz pek olmadığından aşamalar konusunda sıkıntı yaşıyoruz. daemon çalıştırdıktan sonra bi anlaşma evresine geldim fakat sonraki aşamalarda nasıl bi yol izleyeceğimi bilmiyorum.  1tb nvme harddisk ve 32gb ram var

**Comment by de-authority** [comment]
english please.
